tests/python_package_test/test_dask.py

```diff
@@ -826,3 +826,29 @@ def test_dask_classes_and_sklearn_equivalents_have_identical_constructors_except
     assert dask_spec.defaults[:-1] == sklearn_spec.defaults
     assert dask_spec.args[-1] == 'client'
     assert dask_spec.defaults[-1] is None
+
+
+@pytest.mark.parametrize(
+    "methods",
+    [
+        (lgb.DaskLGBMClassifier.fit, lgb.LGBMClassifier.fit),
+        (lgb.DaskLGBMClassifier.predict, lgb.LGBMClassifier.predict),
+        (lgb.DaskLGBMRegressor.fit, lgb.LGBMRegressor.fit),
+        (lgb.DaskLGBMRegressor.predict, lgb.LGBMRegressor.predict),
+        (lgb.DaskLGBMRanker.fit, lgb.LGBMRanker.fit),
+        (lgb.DaskLGBMRanker.predict, lgb.LGBMRanker.predict)
+
+    ]
+)
+def test_dask_methods_and_sklearn_equivalents_have_similar_signatures(methods):
+    dask_spec = inspect.getfullargspec(methods[0])
+    sklearn_spec = inspect.getfullargspec(methods[1])
+    dask_params = inspect.signature(methods[0]).parameters
+    sklearn_params = inspect.signature(methods[1]).parameters
+    assert dask_spec.args == sklearn_spec.args[:len(dask_spec.args)]
+    assert dask_spec.defaults == sklearn_spec.defaults[:len(dask_spec.defaults)]
+    assert dask_spec.varargs == sklearn_spec.varargs[:len(dask_spec.varargs)]
+    assert dask_spec.varkw == sklearn_spec.varkw[:len(dask_spec.varkw)]
```

Comment:
I'm very sorry, I realize now we shouldn't check `.varkw`, since the sklearn functions don't have `**kwargs` pass-through.

https://github.com/microsoft/LightGBM/blob/fc2da2ec84e2dd9aa06421edcb5a625808eb546e/python-package/lightgbm/sklearn.py#L378-L384

Can you please change this check to this? That way, if we add the ability to pass kwargs through `.fit()` or `.predict()` on the sklearn estimators in the future, this test will check that the same name is used with `**` as the Dask estimators use.

```suggestion
    if sklearn_spec.varkw:
        assert dask_spec.varkw == sklearn_spec.varkw
```